frontend/destinations/data/googlecloud.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: internal.odigos.io/v1beta1
+kind: Destination
+metadata:
+  type: googlecloud
+  displayName: Google Cloud Monitoring
+  category: managed
+spec:
+  image: gcp.svg
+  signals:
+    traces:
+      supported: true
+    metrics:
+      supported: false
+    logs:
+      supported: true
+  fields: []
```

Comment:
How does it work without fields??? 🤔

Reply:
In Google Kubernetes Engine there is a feature called Workload Identity that automatically authenticate and authorize Pods to send data to Google Cloud Monitoring, see: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter